README.md

```diff
@@ -1,13 +1,71 @@
+* Open a terminal in the main project directory and run "npm install" (This must be done only once)
+* In the same terminal run "node server"
+* The web server should now be running locally at http://localhost:1337
+
+* With the database running, you can manage its contents easily using MongoDB Compass which can be downloaded at https://www.mongodb.com/download-center#compass
+* With the web service running, you can make HTTP requests to it using Postman which can be downloaded at https://www.getpostman.com/apps
+
+### How to Prepare the Project for Deployment
+
+First, you must set the variables in "config/env/production.js." This lets the web service know information specific to deployment. Set "port" to the port number that this web service will be running on when deployed, eg. 1337. Next, you must set "db" to the address that your MongoDB will be running from, eg. 'mongodb://db1.example.net:27017.' More information about setting this string can be found at https://docs.mongodb.com/manual/reference/connection-string/. Finally, you must set "uiUrl" to the URL that the user interface application will be deployed to, eg. 'gtthriftshop.com' (do not include 'https://' or a '/' at the end).
+
+The next thing that must be done is setting up the secret configuration file. This will contain sensitive information that is specific to deployment. First, make a copy of "secret.config.sample.json" and rename it to "secret.config.json." In this file, set "secret" to any lengthy string of characters and/or symbols of your choice. This will be used when encrypting/decrypting information to make it more secure. Next, set "emailUsername" and "emailPassword" to the email address and the password of the Gmail account that will be used to send account verification and password reset emails. For the Gmail account to be able to send automated emails you must visit https://myaccount.google.com/lesssecureapps and allow less secure apps.
+
+Before deploying the project, it needs a MongoDB database to connect to. The easiest way to do this is through MongoDB's atlas service at https://www.mongodb.com/cloud/atlas. A comprehensive official guide for MongoDB Atlas can be found at https://docs.atlas.mongodb.com/getting-started/.
+
+### How to Deploy
+
+To deploy the project, you can use a cloud platform such as Amazon AWS or Heroku. Heroku has an official step-by-step guide for deploying using the platform. It can be found at https://devcenter.heroku.com/articles/getting-started-with-nodejs. A similar tutorial for deploying to AWS EC2 can be found at https://devcenter.heroku.com/articles/getting-started-with-nodejs.
+
+### Troubleshooting
+
+When running the web service, you may encounter the error message, "Error: Cannot find module." Make sure to install all necessary dependencies by opening a terminal in the main project directory and running "npm install." If the problem persists, delete the folder, "Node Modules," if it exists and try again.
```

Comment:
I think the comma after "folder" and the comma after "Node Modules" should be removed.